tox.ini

```diff
@@ -1,41 +1,45 @@
 [tox]
 envlist =
-    {py3.5,py3.6,py3.7,py3.8}-{sqlite,postgresql}-trytond-{5.0,5.2,5.4,5.6,latest}
+    {py3.5,py3.6,py3.7,py3.8,py3.9}-{sqlite,postgresql}-trytond-{5.0,5.2,5.4,5.6,latest}
     linters
 
 [testenv]
 deps =
     .
-    {py3.5,py3.6,py3.7,py3.8}-postgresql: psycopg2-binary
+    {py3.5,py3.6,py3.7,py3.8,py3.9}-postgresql: psycopg2-binary
     trytond-latest: trytond
     trytond-5.6: trytond>=5.6,<5.7
     trytond-5.4: trytond>=5.4,<5.5
     trytond-5.2: trytond>=5.2,<5.3
     trytond-5.0: trytond>=5.0,<5.1
+    factory_trytond
+    factory_boy
```

Comment:
Isn't factory_boy already present due to being a transitive dep of factory_trytond?

Reply:
That's right